packages/editor-ui/src/components/CodeNodeEditor/CodeNodeEditor.vue

```diff
@@ -262,19 +341,51 @@ export default defineComponent({
 		extensions.push(this.languageCompartment.of(languageSupport), ...otherExtensions);
 
 		const state = EditorState.create({
-			doc: this.modelValue || this.placeholder,
+			doc: this.modelValue ?? this.placeholder,
 			extensions,
 		});
 
 		this.editor = new EditorView({
 			parent: this.$refs.codeNodeEditor as HTMLDivElement,
 			state,
 		});
+
+		// empty on first load, default param value
+		if (!this.modelValue) {
+			this.$emit('update:modelValue', this.placeholder);
+		}
 	},
 });
 </script>
 
 <style lang="scss" module>
+:global(.el-tabs) {
+	:global(.el-tabs__content) {
+		border: 1px solid var(--color-foreground-base);
```

Comment:
these are global changes that will affect `el-tabs` everywhere? does not seem like we have tabs anywhere else? maybe then we can bake these changes into the custom css file for tabs?